conformance-test/libraryMethods.ts

```diff
@@ -795,7 +795,7 @@ export async function notificationGetMetadata(options: ConformanceTestOptions) {
 
 export async function createBucket(options: ConformanceTestOptions) {
   const bucket = options.storage!.bucket('test-creating-bucket');
-  const [exists] = await bucket.exists();
+  const exists = await bucket.exists();
```

Comment:
Whats the change here? previously we were returning an array, and now its an object?
It would be helpful to start a README doc with a list of "breaking" changes so that we make sure we cover everything

Reply:
Or update PR description with a checklist of # of changes that need to happen

Reply:
We were returning arrays for no reason all over the place. I'm trying to bring responses much more in line with the standard JSON API responses.

Reply:
The number of breaking changes is going to be too big for comments / PR commit messsages. I have created an issue in buganizer to create a migration guide.

Reply:
Throwing out some random ideas here... It seems like migrating for returning an array vs object is the big breaking change here. Is there anyway to have some kind of flag for customers who want to keep the current behavior? We can eventually deprecate the flag but it would give customers some more time to keep the current behavior while we modernize the library